themes/src/main/resources/theme/keycloak-preview/account/resources/tsconfig.json

```diff
@@ -12,7 +12,10 @@
     "jsx": "react",
     "suppressImplicitAnyIndexErrors": true
   },
+  "include": [
+    "./**/*.ts?"
+  ],
   "files": [
-    "../../../../../../../../adapters/oidc/js/src/main/resources/keycloak.d.ts"
+    "../../../../../../../../adapters/oidc/js/src/main/resources/keycloak.d.ts",
```

Comment:
A trailing comma isn't allowed in JSON:

```
aszczucz@tyrfing ~> curl -sSL https://raw.githubusercontent.com/keycloak/keycloak/695864d4b70494a0d24f7c54bf5a4cde2e907c45/themes/src/main/resources/theme/keycloak-preview/account/resources/tsconfig.json | jq '.'

parse error: Expected another array element at line 20, column 3
```

Reply:
So what should we do now when it was already merged? Another follow-up JIRA?? @ssilvert